drivers/hud_spm_report/app/controllers/hud_spm_report/base_controller.rb

```diff
@@ -12,13 +14,14 @@ def available_report_versions
       {
         'FY 2020' => { slug: :fy2020, active: false },
         'FY 2023' => { slug: :fy2023, active: false },
-        'FY 2024 (current)' => { slug: :fy2024, active: true },
+        'FY 2024' => { slug: :fy2024, active: false },
+        'FY 2026 (current)' => { slug: :fy2026, active: true },
       }.freeze
     end
     helper_method :available_report_versions
 
     def default_report_version
-      :fy2024
+      :fy2026
```

Comment:
I _think_ we want to leave this until 10/1/2025, maybe a ToDoOrDie? Or maybe just a date-based case statement?  It could become default 9/1/2025 for staging.